### PR TITLE
Bug fix 1221508. Removes duplicated contributors from commit message.

### PR DESCRIPTION
**Pull request by jotes**
@mathjazz @Osmose  r?

### CODE REVIEW

**Comment by mathjazz** [comment]
r+

Plese change commit message to start with "Fix bug 1221508", otherwise the bug won't close. ;)

**Comment by jotes** [comment]
@mathjazz done.

File: pontoon/sync/tests/test_core.py [hunk 1]
```diff
@@ -340,6 +340,26 @@ def test_multiple_authors(self):
...
+
+        commit_changes(self.db_project, self.vcs_project, self.changeset)
+
+        self.repository.commit.assert_called_with(
+            CONTAINS(author.display_name),
+            author,
+            os.path.join(FAKE_CHECKOUT_PATH, self.translated_locale.code)
+        )
+        commit_message = self.repository.commit.mock_calls[0][1][0]
+        assert_equal(len(set(re.finditer(author.display_name, commit_message))), 1)
```
Comment (Osmose) [inline pontoon/sync/tests/test_core.py]: If you wrap it in a set it'll remove the duplicates even if there were any in the message. Also, `re` is overkill for this, you should just be able to do `commit_message.count(author.display_name)`.

**Comment by Osmose** [comment]
r+wc, nice catch and fix! :D

**Comment by mathjazz** [comment]
I'd like to land this today, so I'll merge and fix test in a separate commit before deploying.

**Pull request merged**